<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="Content-Type" content="text/html">
  <meta name="description" content="Compilation of extracted CSS styles on the SCP Wiki">
  <meta name="keywords" content="scp,wiki,styles,wikidot">
  <meta name="author" content="SCP Wiki">
  <meta name="generator" content="wikidot-css-extractor">

  <script src="https://tristen.ca/tablesort/dist/tablesort.min.js"></script>
<script src="https://tristen.ca/tablesort/dist/sorts/tablesort.number.min.js"></script>
<script src="https://ammongit.github.io/wikidot-css-extractor/static/script.js" async></script>
  <link rel="stylesheet" href="https://tristen.ca/tablesort/tablesort.css">
<link rel="stylesheet" href="https://ammongit.github.io/wikidot-css-extractor/static/style.css">

  <title>SCP-6009</title>
</head>
<body>
<div id="container">
  
  <div class="page">
    <h1 class="title">SCP-6009</h1>
    <p>
      Link to article: <a href="https://scp-wiki.wikidot.com/scp-6009" target="_blank">SCP-6009</a>.
    </p>

    <h2 class="header">Module Styles</h2>
    <ul class="module-styles">
      
        <li>
          <pre><code>.newcontainer {
    margin-top: 0.2rem; margin-bottom: -1.5rem;
    display: grid;
    grid-template-columns: 1fr 1fr 1fr 1fr;
    line-height: 20%;
    grid-template-rows: auto;
    grid-template-areas:
        &#34;one two image image&#34;
        &#34;text text image image&#34;;
}
.slot1 { grid-area: one; }
.slot2 { grid-area: two; }
.slot1, .slot2 {
    text-align: center;
    margin-bottom: 0.1rem;
    padding: 4px;
    box-sizing: border-box;
}
.grid-image { grid-area: image; box-sizing: border-box; padding: 0.5rem; }
.textslot { grid-area: text; padding: 0.5rem; padding-left: 0; }

@media (max-width: 767px) {
    .newcontainer {
        grid-template-areas:
            &#34;one one two two&#34;
            &#34;image image image image&#34;
            &#34;text text text text&#34;;
    }
}

.email {border: solid 2px #000000; width: 88%; padding: 1px 15px; margin: 10px; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.emailchiang {border: solid 2px #000000; background: lavenderblush; width: 88%; padding: 1px 15px; margin: 10px auto; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.emailokami {border: solid 2px #000000; background: antiquewhite; width: 88%; padding: 1px 15px; margin: 10px auto; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.evencols tr th,
.evencols tr td {
    width: 280px;
}

.footnotes-footer {
display: none;
}

/*-- fancy collapsible, derived from monkatraz bhl collapsible --*/
div[class*=&#34;email&#34;] .collapsible-block-folded, div[class*=&#34;email&#34;] .collapsible-block-unfolded-link {
    background: #7E2520;
    padding-top: 0.3rem;
    padding-bottom: 0.24rem;
    padding-left: 1rem;
    padding-right: 1rem;
    width: min-content;
    white-space: nowrap;
    overflow: hidden;
    margin:auto; /*-- center --*/
}
div[class*=&#34;email&#34;] .collapsible-block-link {
    color: #f4f4f4;
    font-family: &#39;Fira Code&#39;, monospace;
    font-weight: bold;
    font-size: 96%;
}
div[class*=&#34;email&#34;] .collapsible-block-link:hover {
    text-decoration: none
}

div[class*=&#34;email&#34;] .collapsible-block-content {
    border-top: solid 0.24rem #7E2520;
    border-bottom: solid 0.24rem #7E2520;
}

div[class*=&#34;email&#34;] .collapsible-block-folded .collapsible-block-link::before {
    content: &#34;▷  &#34;;
}
div[class*=&#34;email&#34;] .collapsible-block-unfolded .collapsible-block-link::before {
    content: &#34;▽  &#34;;
}
</code></pre>
        </li>
      
    </ul>

    <h2 class="header">Inline Styles</h2>
    <ul class="inline-styles">
      
        <li>
          <pre><code>display: block ; border : dashed 1px #444444 ; background :#f5f5f5; margin-left : auto ; margin-right : auto ; width : auto ; padding:10px</code></pre>
        </li>
      
        <li>
          <pre><code>font-size: 150%; font-weight: bold; margin: 2px 0 5px;</code></pre>
        </li>
      
        <li>
          <pre><code>width: 100%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-bottom: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>border-right: 1px solid #AAA; text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>text-align: center; width: 25%;</code></pre>
        </li>
      
        <li>
          <pre><code>font-size: 135%; font-weight: bold; margin: 2px 0 5px;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
        <li>
          <pre><code>border:solid 1px #999999; background:#f2f2c2; padding:5px; margin-bottom: 10px;</code></pre>
        </li>
      
        <li>
          <pre><code>display: flex; justify-content: space-between;</code></pre>
        </li>
      
    </ul>

    <h2 class="header">Includes</h2>
    <ul class="includes">
      
        <li>
  
  

  
    <a href="https://scp-wiki.wikidot.com/theme:paperstack" target="_blank"><code>:scp-wiki:theme:paperstack</code></a>
  

  
    
  <span class="page-info">
    [<a href="/wikidot-css-extractor/pages/theme:paperstack.html" target="_blank">info</a>]
  </span>

  
</li>
      
        <li>
  
  

  
    <a href="https://scp-wiki.wikidot.com/component:toggle-sidebar" target="_blank"><code>:scp-wiki:component:toggle-sidebar</code></a>
  

  
    
  <span class="page-info">
    [<a href="/wikidot-css-extractor/pages/component:toggle-sidebar.html" target="_blank">info</a>]
  </span>

  
</li>
      
        <li>
  
  

  
    <a href="https://scp-wiki.wikidot.com/component:image-block" target="_blank"><code>component:image-block</code></a>
  

  
</li>
      
        <li>
  
  

  
    <a href="https://scp-wiki.wikidot.com/component:license-box" target="_blank"><code>:scp-wiki:component:license-box</code></a>
  

  
    
  <span class="page-info">
    [<a href="/wikidot-css-extractor/pages/component:license-box.html" target="_blank">info</a>]
  </span>

  
</li>
      
        <li>
  
  

  
    <a href="https://scp-wiki.wikidot.com/component:license-box-end" target="_blank"><code>:scp-wiki:component:license-box-end</code></a>
  

  
    
  <span class="page-info">
    [<a href="/wikidot-css-extractor/pages/component:license-box-end.html" target="_blank">info</a>]
  </span>

  
</li>
      
    </ul>

    <h2 class="header">CSS Classes</h2>
    <ul class="css-classes">
      
        <li>
          <pre><code>newcontainer</code></pre>
        </li>
      
        <li>
          <pre><code>slot1</code></pre>
        </li>
      
        <li>
          <pre><code>slot2</code></pre>
        </li>
      
        <li>
          <pre><code>grid-image</code></pre>
        </li>
      
        <li>
          <pre><code>textslot</code></pre>
        </li>
      
        <li>
          <pre><code>evencols</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailokami</code></pre>
        </li>
      
        <li>
          <pre><code>emailchiang</code></pre>
        </li>
      
        <li>
          <pre><code>blockquote</code></pre>
        </li>
      
        <li>
          <pre><code>footer-wikiwalk-nav</code></pre>
        </li>
      
    </ul>

    <h2 class="header">Page Source</h2>
    <pre><textarea class="page-source" readonly>[[include :scp-wiki:theme:paperstack]]
[[include :scp-wiki:component:toggle-sidebar]]

[[module CSS]]
.newcontainer {
    margin-top: 0.2rem; margin-bottom: -1.5rem;
    display: grid;
    grid-template-columns: 1fr 1fr 1fr 1fr;
    line-height: 20%;
    grid-template-rows: auto;
    grid-template-areas:
        &#34;one two image image&#34;
        &#34;text text image image&#34;;
}
.slot1 { grid-area: one; }
.slot2 { grid-area: two; }
.slot1, .slot2 {
    text-align: center;
    margin-bottom: 0.1rem;
    padding: 4px;
    box-sizing: border-box;
}
.grid-image { grid-area: image; box-sizing: border-box; padding: 0.5rem; }
.textslot { grid-area: text; padding: 0.5rem; padding-left: 0; }

@media (max-width: 767px) {
    .newcontainer {
        grid-template-areas:
            &#34;one one two two&#34;
            &#34;image image image image&#34;
            &#34;text text text text&#34;;
    }
}

.email {border: solid 2px #000000; width: 88%; padding: 1px 15px; margin: 10px; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.emailchiang {border: solid 2px #000000; background: lavenderblush; width: 88%; padding: 1px 15px; margin: 10px auto; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.emailokami {border: solid 2px #000000; background: antiquewhite; width: 88%; padding: 1px 15px; margin: 10px auto; box-shadow: 0 1px 3px rgba(0,0,0,.5)}

.evencols tr th,
.evencols tr td {
    width: 280px;
}

.footnotes-footer {
display: none;
}

/*-- fancy collapsible, derived from monkatraz bhl collapsible --*/
div[class*=&#34;email&#34;] .collapsible-block-folded, div[class*=&#34;email&#34;] .collapsible-block-unfolded-link {
    background: #7E2520;
    padding-top: 0.3rem;
    padding-bottom: 0.24rem;
    padding-left: 1rem;
    padding-right: 1rem;
    width: min-content;
    white-space: nowrap;
    overflow: hidden;
    margin:auto; /*-- center --*/
}
div[class*=&#34;email&#34;] .collapsible-block-link {
    color: #f4f4f4;
    font-family: &#39;Fira Code&#39;, monospace;
    font-weight: bold;
    font-size: 96%;
}
div[class*=&#34;email&#34;] .collapsible-block-link:hover {
    text-decoration: none
}

div[class*=&#34;email&#34;] .collapsible-block-content {
    border-top: solid 0.24rem #7E2520;
    border-bottom: solid 0.24rem #7E2520;
}

div[class*=&#34;email&#34;] .collapsible-block-folded .collapsible-block-link::before {
    content: &#34;▷  &#34;;
}
div[class*=&#34;email&#34;] .collapsible-block-unfolded .collapsible-block-link::before {
    content: &#34;▽  &#34;;
}

[[/module]]


[[&gt;]]
[[module Rate]]
[[/&gt;]]

[[div class=&#34;newcontainer&#34;]]

[[div class=&#34;slot1&#34;]]
[[size 90%]]##grey|**ITEM #:**##[[/size]]

[[size 180%]]**SCP-6009**[[/size]]
[[/div]]

[[div class=&#34;slot2&#34;]]
[[size 90%]]##grey|**OBJECT CLASS:**##[[/size]]

[[size 180%]]**TICONDEROGA**[[/size]]
[[/div]]

[[div class=&#34;grid-image&#34;]]
[[include component:image-block name=http://scp-wiki.wdfiles.com/local--files/scp-6009/main.jpg|caption=6009-Catena surrounding a neuron (2500x) |width=300px|align=center]]
[[/div]]

[[div class=&#34;textslot&#34;]]


**Special Containment Procedures:** Due to the necessity of SCP-6009 for biological function, no containment procedure so far has been found to be successful, and is likely impossible and unnecessary. Containment procedure currently focuses on prevention of information leak regarding SCP-6009 and the 6009-Catena region.

The microscopic size of SCP-6009 and lack of immediately noticeable anomalous traits makes it unlikely a civilian researcher would uncover the anomaly. Publications regarding genetics, epigenetics, microbiology, and neuroscience should be screened for papers pertaining to SCP-6009, under purview of the Nanobiomics Department whether to allow publication.

Medical textbooks have been standardized to include the 6009-Catena region as protective tissue, which can be safely ignored. Because of this, it is unlikely a civilian physician or surgeon would be able to deduce the anomalous nature of 6009-Catena; however, medical journals and case reports of neurosurgery should still be screened for mentions of 6009-Catena or SCP-6009.

[[/div]]


[[/div]]

@@@@

Genomic data regarding cluster association obtained from Project Vigenère are available upon request for all personnel of Clearance Level 3 and above. All inquiries and testing regarding SCP-6009 are under the jurisdiction of the Nanobiomics Department.

A proposal to reclassify SCP-6009 from Ticonderoga[[footnote]] As per the Foundation [https://scp-wiki.wikidot.com/esoteric-classes-complete-list#toc0 Guide on Esoteric Object Classes]: Ticonderoga is for “Object class for anomalies that cannot be contained, but do not need to be contained.” [[/footnote]] to Thaumiel is on hold, pending the results of Project Vigenère.

@@@@

**Description:** SCP-6009 is the human mitochondrion.

Structurally and functionally, SCP-6009 are largely similar to mitochondria in other eukaryotes, including its role in cellular respiration, cell cycle regulation, and certain signalling pathways. A number of factors differentiate SCP-6009 from mitochondria found in other eukaryotes.

@@@@

[[div class=&#34;evencols&#34;]]
||~ Category ||~ Non-anomalous mitochondrion ||~ SCP-6009 ||
|| Image || [[image http://scp-wiki.wdfiles.com/local--files/scp-6009/mito1 width=&#34;250px&#34;]]|| [[image http://scp-wiki.wdfiles.com/local--files/scp-6009/mito2 width=&#34;250px&#34;]] ||
|| Physical Size || ~500-1000 nm || ~200-500 nm ||
|| Motility (in-cell) || Achieved by attaching to the motor/adaptor complex (MIRO1) || Achieved by attaching to the motor/adaptor complex (MIRO1) ||
|| Motility (out-cell) || No such behavior observed || Observed, mechanism unknown ||
|| Mitochondria DNA (mtDNA) Size || ~16.5 kbp in animals, ~2 kbp in plants || ~16-18 kbp depending on location ||
|| Existence Outside of Cells || Rare occasions in the blood stream || See addendum on 6009-Catena ||
|| Fusion and Fission || Very common || Very common ||
[[/div]]

**Addendum - 6009-Catena:** The most notable anomaly surrounding SCP-6009 that differentiates it from conventional cell biology is the existence of the 6009-Catena region. SCP-6009 is observed to exist outside of cells and congregate into colony-like structures, and exhibit high levels of motility, the mechanism of which does not conform to motile mechanisms used within the cell. SCP-6009 instances are observed to sometimes fuse into chain-like structures in the 6009-Catena region; this behavior is not observed elsewhere in the body. 

The 6009-Catena region is a web of these SCP-6009 chains along the extracellular matrix, found interweaved in between the dendrites and myelinated axons of the brain. The vast majority of SCP-6009 in 6009-Catena surrounds the basal ganglia, but strands of 6009-Catena extend within all areas of the neural circuitry. The vast majority of 6009-Catena does not move much, but the ends of the &#34;ladders&#34; are motile and free-moving, and is believed to used to maintain the shape and connection of the region. Proteins, certain ions, and neurotransmitters flow freely in and out of 6009-Catena and neurons, creating an alternative signaling mechanism to maintain function. While it only constitutes roughly 2% of white matter by mass, it is believed to have a significant role in neurological function and neural circuitry formation. 

The anomalous nature of SCP-6009, as differing from other eukaryotes, did not become clear until the discovery of 6009-Catena. As SCP-6009 typically also perish within hours after the death of its host human, posthumous autopsies have not detected SCP-6009 activity in 6009-Catena regions. SCP-6009, within a cultured cell //in vitro//, behaves exactly the same as non-anomalous mitochondrion, which is why it has eluded researchers for years. SCP-6009 activity is most prevalent within neurons and outside neurons in the 6009-Catena region.

Holistic whole-mitochondrial DNA sequencing[[footnote]] As opposed to conventional mitochondrial sequencing, or whole genome sequencing [[/footnote]] using ChOMP-Seq revealed an important discrepancy that is the focus of Project Vigenère. The vast majority of mtDNA found in SCP-6009 are well-understood, and codes for non-anomalous proteins such as NADH dehydrogenase, part of regular mitochondrial function. As in regular non-anomalous mitochondria, much of SCP-6009&#39;s proteins are synthesized from nuclear genes[[footnote]] Genes in the nucleus [[/footnote]]. However, where heteroplasmy was thought of as an unintended natural result of mitochondrial replication, it is now believed to be key to SCP-6009 function. The function and distribution of the unknown mitochondrial genes is the subject of Project Vigenère. It is believed that these genes are responsible for the formation of 6009-Catena, as well as other anomalies such as observed motility and interference with neural circuitry.

Much of what has been understood about SCP-6009 was discovered in a collaboration between Dr. Okami Ryōsuke (formerly of the Neuroscience Department) and Dr. Chiang Wei-Huo (formerly of the Genetics Department), both considered founding members of the Nanobiomics Department.


[!-- Note to all translators: There is a blank version of mapping.jpg in the files --]


[!-- Note to translators from -CN, -ZH, and -JP: Please use the following names (or translate from them): 

Chiang Wei-Huo: 蔣渭火
Okami Ryōsuke: 岡見亮輔
Matsunaga Yuko: 松永裕子
Hsieh Jin-Yan: 謝金雁
Yamataka: 山高

You can change their nickname to whichever makes more sense and sounds better, don&#39;t need to stick with &#34;Waley&#34; or &#34;Ryo&#34; if it sounds awkward --]

**Archived Project Proposal:** Filed August 29, 2010 (rejected); September 4, 2010 (rejected); September 8, 2010 (rejected); September 20, 2010 (rejected); October 1, 2010 (accepted)

[[div style=&#34;display: block ; border : dashed 1px #444444 ; background :#f5f5f5; margin-left : auto ; margin-right : auto ; width : auto ; padding:10px&#34;]]
[[=]]
------

[[div style=&#34;font-size: 150%; font-weight: bold; margin: 2px 0 5px;&#34;]]
[[=]]
{{Cross-Departmental Collaboration Request}}
[[/=]]
[[/div]]
------
[[table style=&#34;width: 100%;&#34;]]
[[row]]
[[cell style=&#34;border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 90%]]**Requester**[[/size]]
[[/cell]]
[[cell style=&#34;border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 90%]]**Originating Department**[[/size]]
[[/cell]]
[[cell style=&#34;border-bottom: 1px solid #AAA; border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 90%]]**Target Department**[[/size]]
[[/cell]]
[[cell style=&#34;border-bottom: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 90%]]**Project Name**[[/size]]
[[/cell]]
[[/row]]
[[row]]
[[cell style=&#34;border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 80%]]Chiang Wei-Huo [[/size]]
[[/cell]]
[[cell style=&#34;border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 80%]]Genetics Department[[/size]]
[[/cell]]
[[cell style=&#34;border-right: 1px solid #AAA; text-align: center; width: 25%;&#34;]]
[[size 80%]]Neuroscience Department[[/size]]
[[/cell]]
[[cell style=&#34;text-align: center; width: 25%;&#34;]]
[[size 80%]] - [[/size]]
[[/cell]]
[[/row]]
[[/table]]

[[=]]

------

[[div style=&#34;font-size: 135%; font-weight: bold; margin: 2px 0 5px;&#34;]]
**{{Request Details}}**
[[/div]]
[[/=]]
[[&lt;]]

Preliminary research into the SCP-6009 genome and 6009-Catena has suggested requiring further research. With this collaboration, we hope to understand:

1. The reason and method of motility of SCP-6009: It&#39;s been observed to be mobile throughout the body, and is key to how they are able to form 6009-Catena, but the method for extracellular and intercellular movement does not follow the well-understood intracellular motility. 

2. The reason and method of 6009-Catena formation: It&#39;s clear that because of the role mitochondria play in cell function, there&#39;s constant interactions between the 6009-Catena and the neurons it surrounds, but we do not know the details of this interaction. We also might try to investigate if we can find any patterns in how 6009-Catena is formed.

As a crucial element of SCP-6009 is the interaction of 6009-Catena, as well as its association and unknown function, we require expertise in neuroscience to better model 6009-Catena and its possible interaction with the brain.


@@@@
[[/&lt;]]
------
[[/=]]
[[/div]]

@@@@
@@@@

**Archival Addendum:** The following communications have been archived pertaining the early research into SCP-6009 from Project Vigenère. Please note that they may contain outdated or inaccurate information.


@@@@

@@@@

[[=]]
[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
October 5, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Greetings Dr. Chiang Wei-Huo,

My name is Okami Ryōsuke, I am a Level 3 Researcher working in Site-84 under the Neuroscience Department. They have assigned me to work with you on your project. I have reviewed the material you sent over about SCP-6009, and your hypothesis regarding the formation of 6009-Catena.

I&#39;ve been studying neuroscience for a long time and this is the first time I&#39;ve ever heard of this sort of structure in the brain. So clearly the first question is: How could thousands of scientists have missed this?

I know you submitted a cross-department collaboration request because you think someone in the Neuroscience Department would be able to assist you in modelling 6009-Catena. Conventional MRI and PET scans are enough to map out the white matter in the brain, which is where most of 6009-Catena is attached to, so what needed to be modified? Please elaborate.

Please respond at your briefest convenience.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
October 20, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Dr. Okami,

Good to have you on board! My colleagues at the Genetics Department don’t quite have the same faith in the SCP-6009 project.

I came across 6009-Catena by chance; my team were studying [[[SCP-3966]]] and found microscopic web-like structures for freshly diseased victims. We thought it was related, but extensive examination on deceased D-Class that was unrelated to SCP-3966 cleared the air. The structure is only visible under TEM[[footnote]] Transmission electron microscopy [[/footnote]]because it&#39;s so miniscule, so it was hard to spot, but once you know where to look it&#39;s everywhere.

We tried to get it replicated in the lab, it doesn&#39;t ever seem to work. And, obviously, mitochondria shouldn’t exist on its own outside of a cell, which is why this whole situation is confusing even without the ladder structure they make. 

Based on preliminary fluorescent imaging it&#39;s spread throughout the brain&#39;s white matter, and has its own connection scheme; they form sort of miniscule nodes at some neural junctions. I don&#39;t know neuroscience enough to make any sense of it. 

Also based on the imaging, it moves in the body on its own, not by kinesin or any of the usual suspects. It kind of wriggles in and out of the brain, but it&#39;s most active in 6009-Catena; it sort of forms a web of nodes. We obviously can&#39;t observe it in living human beings, but you can still sort of get a sense of what it does a few hours within a person&#39;s death.

I suspect the motility has something to do with the less common genes that have been overlooked because heteroplasmy was thought of as basically genetic junk. That&#39;s probably the first thing I should rule out before going onto more wild ideas.

I&#39;m currently trying see if there&#39;s any sequencing protocols that can give us that information. Whole-genome sequencing kind of comes close, but it doesn&#39;t really quite do the trick, any suggestions?

The two main anomalies of SCP-6009 aren&#39;t earth-shattering, but they certainly don&#39;t fit any current understanding of chemistry or biology; I think digging deeper into what might be the scientific mechanism how SCP-6009 forms 6009-Catena, we might be able to pinpoint the exact origin? 

I think both motility and the Catena formation are intertwined with each other, so hopefully solving one solves the other. 

If you aren&#39;t interested in the collaboration, it&#39;s alright if you decline - you aren&#39;t the first person that I&#39;ve been connected with, I&#39;ve submitted this collaboration request to the Neuroscience Department a few times. Even the one who originally first saw the web-like structure, Dr. Jenny Hsieh, who also is a neuroscientist, says it isn&#39;t worth investigating further, but I think we&#39;re onto something.

[[/&lt;]]
[[&gt;]]
Chiang
[[/&gt;]]
[[/div]]



[[=]]
[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
November 12, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Greetings Dr. Chiang Wei-Huo,

I’m not very well-versed in cell biology, and this might not make sense, but is it possible that the heteroplasmic difference you mentioned in the preliminary report is a possibility of the anomaly that arises from SCP-6009? A paper that I read the other day about how they recently discovered certain introns that are removed during RNA splicing, previously thought to be useless may have a role in mRNA degradation.

Is such a method even possible? I know a single mitochondrion contains hundreds of copies of circular DNA, and accounting for the tiny percentage that is abnormal would be a nightmare. Whole-genome sequencing already handles mitochondrial DNA, so what&#39;s your new approach?

I&#39;ve met Dr. Hsieh a few times, but I am not very familiar with her.

As for the collaboration, I am currently not working on any important projects, so I am intrigued by this proposal.

I&#39;m not entirely convinced that we can do much with SCP-6009. I&#39;m still a bit skeptical, but I&#39;m still happy to collaborate further - does the project have a name?

Please respond at your briefest convenience.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
November 29, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Dr. Okami,

Wonderful! I am very glad to hear someone is finally willing to take on my pet project.

I&#39;ve named this Project Vigenère, after the cipher - after all, we are decoding a mystery without knowing what&#39;s waiting for us in the end! Cryptography fascinates me, but I&#39;m awful at computer science. Genetics is basically biological cryptography anyways.

Anyhow, I&#39;ve attached some information on how you can see 6009-Catena from brain tissue samples. Haven&#39;t found a way to do it in living humans, but hey, that&#39;s where you come in!
[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
++ **__Project Vigenère: Visualizing 6009-Catena__**
Dr. Chiang Wei-Huo
[[/=]]

[[include component:image-block
| name=http://scp-wiki.wdfiles.com/local--files/scp-6009/ladder
| caption=Chain-like structure found in 6009-Catena (25000x)]]


SCP-6009 is both integral to cell function, and dependent on cytoplasmic elements to maintain active. Soon after brain death, 6009-Catena rapidly loses structural coherence and would be indistinguishable from somatic SCP-6009. To visualize 6009-Catena, a brain biopsy should be performed on subjects that have been dead for less than one hour. Samples of white matter are where the nodes and chains of 6009-Catena are most likely to be found.

There is currently no way to keep the 6009-Catena structure intact in a sample. Freezing has proven to disrupt the cohesion of 6009-Catena. Conventional culture broth such as Luria broth have shown to slow the decay to up to two hours, but do not stop it. Dye-assisted staining can be used for color contrast, but it is not recommended for samples that are intended to undergo further testing.

6009-Catena both has chain-like components and free-moving components. The chain-like components are much harder to visualize as they are the first to decay.  Transmission electron microscopy (TEM) is the only way to visualize the ladder structure of 6009-Catena in a sample.

@@@@

[[/div]]

[[/collapsible]]

You should be able to see it once you know what you&#39;re looking for; I don&#39;t think anyone ever realized such a structure existed within our brains.

Also, please call me Waley! Saying my whole name sounds like I&#39;m some dead politician.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
December 2, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

Very well then, nice to meet you. You may call me Ryo.

I followed what you did - quite remarkable, but so delicate! Why did no one want to work on this? This is quite the medical discovery.

Regarding mapping the 6009-Catena region: conventional MRI and PET scans do the trick in mapping white matter, which is where most of where SCP-6009 congregate to form 6009-Catena. I think I&#39;ll figure something out with cytoplasmic exclusion so we can get it going.

My colleague, Dr. Michael Aguinaldo, indicates that he may have some idea from his involvement in the early stages of the Khevtuul probe project. The [[[SCP-2669]]] file is restricted though, so Dr. Aguinaldo says he needs to check in with Director Yamataka for clearance before involving him in our project.

The World Connectome Project just recently released their latest tractography model, but it’s not accurate enough for our purposes and misses out on the inter-neural space where most of 6009-Catena resides. I&#39;ll look more into it though.  

Do you have MRI machines on your site? Ours just broke, again, so I have to take the train to Osaka every day. Knowing Site-84, it&#39;s going to just stay broken forever.

Please respond at your briefest convenience.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]
[[/=]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
December 28, 2010
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Ryo,

That sounds great! Does Dr. Aguinaldo want to join our team?

Here&#39;s a rough summary of what I named ChOMP-Seq. It&#39;s based off an abandoned side project when we were studying how the genome absorption effect of [[[SCP-2946]]] worked... didn&#39;t think I&#39;d find use for it.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
++ **__Project Vigenère: Analyzing anomalous mitochondrial DNA using ChOMP-Seq__**
Dr. Chiang Wei-Huo • Dr. Okami Ryōsuke
[[/=]]

A working hypothesis for Project Vigenère aims to look at the full genomic structure of SCP-6009 to understand the roots of its anomaly.

The Chiang-Okami Multi-Plasmid Sequencing (ChOMP-Seq) improves on conventional sequencing techniques for circular DNA, which typically have the drawback of needing specific primers and requiring PCR amplification, which disrupts the heteroplasmic ratio that is hypothesized to be a root of the anomalous nature of SCP-6009. ChOMP-Seq is a modification of the Circular DNA enrichment sequencing (CIDER-Seq), and uses Lysenko primers instead to ensure equal amplification before PCR. A difference of ChOMP-Seq over conventional whole genome sequencing (WGS) is maintaining the quantitative occurrences of non-nuclear DNA. ChOMP-Seq results are stored as custom CSQ files to retain the quality score and ratio.

ChOMP-Seq has uncovered a new paradigm of genetics. Much of the SCP-6009 DNA is similar to their non-anomalous counterparts and code for non-anomalous proteins integral to cellular function. While it was previously believed the entire mitochondrial DNA was mapped, ChOMP-Seq revealed that conventional sequencing ignored heteroplasmy and thus only accounted for the most common circular DNAs found in SCP-6009.

Project Vigenère is the full analysis of sequences found, and in particular the differences of behavior of SCP-6009 in somatic cells, neurons, and the 6009-Catena region. The project will take into account chromosomal DNA, mapped connectomes, as well as cell-to-cell interactions to paint a full picture.

The two key anomalous properties we hope to decipher is the issue of motility, and the issue of the formation of 6009-Catena. We believe that the genomic code is key to understanding.

@@@@

[[/div]]

[[/collapsible]]

As you can see, a pretty nifty way of sequencing. It seems to work, but because of the parallel sequencing and increased scrutiny during the amplification stage, it gives us a lot of useless data.

I see your site is having problems too... our equipment is fine, but our site has literally only been allotted only four D-class personnel. To run some preliminary comparisons and sequence alignments, I had to take the DNA of some of the junior researchers as well as my own, just so I can see whether ChOMP-Seq actually worked.

Hopefully my request for more D-class goes through, but I doubt it.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
Janurary 5, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]


[[&lt;]]

Dear Waley,

I see, that&#39;s quite clever.

You&#39;re very kind! ChOMP-Seq is a nice and catchy name. I don&#39;t know if I feel good about being in the acronym of it though - I only contributed a small suggestion, it was mainly based off of your work and you did all of the testing.

Is there any way for ChOMP-Seq it to output something more standard like FASTQ or VCF, or even just a .txt file? The computers at Site-84 are really outdated and just can&#39;t seem to convert or read your custom CSQ files.

Dr. Aguinaldo has other projects he&#39;s working on, but he&#39;s willing to assist me in shaping up the 6009-Catena model. He got clearance to use his previous work, so we are good! You should now see some Catena networks from D-Class personnel on our drive, check and see if that&#39;s what you wanted. Each file is massive, by the way, but that’s unavoidable. The scan takes around twenty minutes, but generating the model afterwards takes our computers all afternoon. Well, it wouldn&#39;t need to take all afternoon if my computers weren&#39;t from twenty years ago. 

Please respond at your briefest convenience.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]
[[/=]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
Janurary 20, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Ryo,

Coming up with fun names and acronyms is kind of how I spend my spare time... kind of pathetic, right? Plus your name starts with a vowel, and that opens up so many windows for naming schemes. ChOMP-Seq is just so much more catchy that any random assortment of letters. And yes, FASTQ probably would work, you should update to the newest version now.

I saw the neural map files - I think you need to take into account node distribution after the initial scan, but the tractography looks exactly like what I want.

Also, I don&#39;t mean to offend, but I don&#39;t think &#34;briefest convenience&#34; is right? Do you mean &#34;earliest convenience&#34;?
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
February 20, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

My English isn&#39;t very good... I have been saying that for years because I thought it was proper and polite. No one ever corrected me on that... I rarely get embarrassed like this, so that was not a pleasant feeling for me. 

As we previously discussed, here is a schema to explain 6009-Catena mapping. I&#39;m showing this to Director Yamataka to get him to allot some funds for our project... for how big and mighty the SCP Foundation is, it&#39;s always so painful to squeeze money out of them. I don&#39;t envy the life of a Foundation accountant.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]

[[=]]
@@@@
++ **__Project Vigenère: Neural Mapping of 6009-Catena__**
Dr. Okami Ryōsuke • Dr. Michael Aguinaldo

[!-- Note to translators: There is a blank version of mapping.jpg in the files --]

[[image
http://scp-wiki.wdfiles.com/local--files/scp-6009/mapping.jpg]]
[[/=]]

Modelling the 6009-Catena hodological network requires two components: an accurate distribution of SCP-6009 nodes, and an accurate neural tractogram.

An accurate distribution of SCP-6009&#39;s nodes combines an conventional diffusion tensor imaging (DTI) segmentation, with a modified T1-weighted magnentic resonance imaging (MRI) to obtain a physical structure of neuron location; radiofluorescence-tagged SCP-6009 used for comparison of the DTI segmentation gives a relatively accurate distribution for SCP-6009 node structure, after cytoplasmic exclusion to map the extracellular space.

Connectome tractography gives the physical neuron relationships of a subject&#39;s brain. Much of conventional tractography are estimations of truth bundles based on diffusion MRI imaging, which has a high incidence of invalid connections. Khevtuul replication requires an accurate copy of the mind, but does not sort out the connections in a clear matter. By taking an unanchored exampled of a Khevtuul replica, we are able to directly transfer with no physical barrier, thus producing an accurate tractogram.  

By combining the node distribution of SCP-6009 combined with the connection tratogram, a hodological network can be determined. It takes roughly 4 hours to completely compute the hodological network, due to high incidence of false positives.


[[/div]]
[[/collapsible]]

I made the diagram myself. It is, sadly, a bit rough, but I think this gets the point across. I sent it to our graphics team, but I do not expect them to get to me in time.

The new update to the ChOMP-Seq software fixed my issue, how did you do it? I thought you didn&#39;t know anything about computers?
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
February 27, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Hey, I&#39;m not entirely inept, I do know SOME things.

@@@@

@@@@

Ok, fine, you caught me. Jenny fixed it.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@



[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
March 1, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Ryo,

How did your meeting with Yamataka go?

It can&#39;t be that much of a hassle to get an MRI fixed right?
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
March 7, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

Yes, Director Yamataka allotted the funds, but he actually paid attention to my presentation and was surprisingly attentive. I never saw him be so interested in anything I&#39;ve worked on. Truth be told, I had my suspicions that he gave me your project because he expected it to be a dud.

He gave me some pointers about ways to improve the model, but also asked if we have been thinking about genes associated with the signalling pathways, that might cause either the motility of SCP-6009 or the formation of 6009-Catena. I don&#39;t think we have thought about that angle right? Because the signaling pathway is not possible to be observed //in vitro//, we tried. Every time we try to culture them they die; they might have originated from bacteria, but they need like, cytoplasmic ribosomes and nuclear genes to actually function. The Catena web is definitely affecting how they interact with neurons.

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
March 10, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Funny that he would mention that, because I was actually discussing this with Jenny... she wouldn&#39;t tell me what she&#39;s working on, but she mentioned the same thing about chemical pathways. And actually, she suggested that I put out a request to collaborate with the Nanotechnology Department.

She came up with this idea of an //in vivo// neural observation nexus... I don&#39;t really know what she means, but that sounds cool as hell so I&#39;m on board.

I&#39;ll keep you updated.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
March 25, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

So, funny story, Ryo. I saw that a &#34;Dr. Matsunaga Yuko&#34; was approved to work with us and thought she&#39;s going to work with you but apparently... no, she&#39;s here in Taiwan. And apparently she&#39;s been at Site-168 for two years now and I&#39;ve never met her? Interesting, our site doesn&#39;t have that many people. 

Makes sense she&#39;s here since our site is a nanotech hub, but I just thought it was funny.

We&#39;ve only been working together for a week and she already made massive improvements on the protein-based probe idea. I&#39;ve been looking into some chlorosomes antenna complexes from green sulfur bacterium... we&#39;ll see where that takes me.

She&#39;s amazing at what she does, Ryo. I think you&#39;d love to meet her.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 15, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

I&#39;m sure I&#39;ll see her one day.

I&#39;ve got some important news: so Yamataka came the other day and said that the Biological Sciences Summit is in Chengdu in like a month, at the beginning of May. He said he wants me to present our work on Project Vigenère, he might be able to get us an hour slot. He wants our teams to meet, so me and Dr. Aguinaldo will be there if you&#39;re joining.

I think he just wants us to present the 6009-Catena Hodological Network, but I think you said Matsunaga had promising stuff to show? If we can showcase that then it will truly be impressive.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]



[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 18, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Oh, that would be wonderful if Director Yamataka could get us a spot, I always wanted to visit Sichuan.

I think the probe-nexus design we have been working on... kind of works? We got a lot of promising results from the D-Class personnel that we&#39;ve been getting. Matsunaga worked some magic with Site Director Chen and suddenly we have like, 40 of them. Miracle worker, that one.

When I say &#34;kind of works&#34; I mean, it gets us the result we want, but it has a giant drawback, as you can see.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]
[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@

++ **__A New Method of Molecular Observation: The MH Nexus and CMN probe__**
**Dr. Matsunaga Yuko • Dr. Chiang Wei-Huo • Dr. Hsieh Jin-Yan**
[[/=]]

[[&lt;]]
The revolutionary design based on the works of Dr. Matsunaga and Dr. Hsieh solves the problem of data collection on a cellular basis. The design works twofold: distributed nanosensory probes (&#34;Chiang-Matsunaga Nanosensory Probe&#34;, or CMN probe) and a centralized, per-subject collection center (&#34;Matsunaga-Hsieh nexus&#34;, or MH nexus) that utilizes a dual-channel design that allows signal transmission out of the body. The probes utilizes a modified version of the Fenna–Matthews–Olson Complex; the chlorosomal nature both allows non-conformational attachment to various ion channels, and the issue of quantum Zeno freeze is abated by utilizing excitonic Anderson localization. The quantum resonance profile of the molecule passing through the center of the CMN probe is able to be transmitted to the MH nexus. The MH nexus then transmits the data physically through an antenna-like wire and allows data transmission outside the body.

[[include component:image-block
| name=http://scp-wiki.wdfiles.com/local--files/scp-6009/nexus
| caption=MH Nexus in D-class subject]]

Due to the massive amounts of raw data generated, //in vivo// data storage is unsustainable, thus necessitating a physical wire acting as an exit port, which crosses the meninges. While this has proven to be the most direct and fastest way data can be transmitted outside of the cranium, it also poses significant risks for post-implantation surgical recovery.

The CMN probes are currently unable to self-replicate, but it is cost effective to manufacture at a large scale. Sufficient data can be obtained at around 4,000 probes per nexus.

One nexus outputs around 1.5 terabytes of data per minute; that means it’s impossible to store, and physical transmission is the only method we have to accurately and reliably obtained the data. That means the bulky physical wires whose only job was to transmit.

Because of the distance and size of 6009-Catena, and difficulty regarding signal transduction across flesh without affecting it, there are two reasons for concern. Due to the location of the nexus and the exit ports, post-surgery there is always the degree of swelling, and 55% D-Class personnel experience great periods of migraines afterwards. In addition, certain parts of the neural circuitry and associated 6009-Catena region would be affected physically, which introduces a unwanted variable into the study. More research in this area is needed.

[[/&lt;]]
[[/div]]

[[/collapsible]]

We&#39;re trying to improve on the MH nexus design more, but the biggest problem is still signal transduction. The wire is causing all of the problems, since it&#39;s basically an open wound in your head and all the D-class complain it&#39;s really uncomfortable. It turns there&#39;s really no way to easily transmit terabytes worth of data over delicate flesh and bone!

Not a big fan of the acronyms, but since Matsunaga was the hero of the day I&#39;m fine with her getting the naming rights.

You should see the relevant files on our server soon. And yes, we&#39;re all going to the symposium.

You know, we&#39;ve been collaborating for a few months now, just now it hit me I don&#39;t even know what you really look like... I&#39;m sure you look better in person than in that cold Foundation profile picture.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 27, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

Yamataka looked very shocked.

I will take that as a yes.

See you in Chengdu! 
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@



[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
May 14, 2011
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

Sorry I was holed up in the convention center all week, spicy food really does a number on me and I didn&#39;t want to risk it. Maybe next time I&#39;ll join you next time when we&#39;re in a city safer for my stomach. Plus I&#39;m not an adventurous spirit in a country I don&#39;t speak the language of.. but parts of the old town looked beautiful!

At any rate, while you were on your urban adventure, Yamataka actually brought me and Dr. Matsunaga and introduced us to Dr. Everett Mann.

Dr. Mann was very intrigued by our probe project, and he had a pretty riveting discussion with Matsunaga on probe design and implantation. However, he specifically sought us out to talk about Project Vigenère. He saw our presentation, Waley!

Dr. Mann have taken interest in our project, he thinks what we&#39;re doing might lead to something important. He actually has taken up on himself to request the formation of a new department just for this project, and wants us to know if we&#39;re willing to do it.

In addition, he said that our mode of collaboration was noble but very inefficient. He said he&#39;s honestly impressed we got so far by throwing emails at each other and concurrently working at two underfunded sites: admirable but not efficient at all. Work smarter, not harder, he said. He said it would be better for us if we were at the same site, and he had sent out a notice to some site directors he knew. You might get some offers soon.

There&#39;s probably also going to be more people joining us, since we&#39;re a going to be a proper department. New departments always get a lot of buzz.

He said that he put in a request for “Department of Mitochondrial Studies” but even a bore like me thinks that is a stupid name. I&#39;m not as creative as you are, can you come up with something punchy?

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 1, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-168
Hsinchu, Taiwan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Oh, wow, that&#39;s... a lot to take in at once. And of course, I already said yes to department formation! Wow, Dr. Mann, out of all people, becoming our benefactor. 

I got a bunch of offers, but the only one that was interesting was from Director Leep Andrews of Site-234, who reached out to me after he heard about our team looking to become a department. He said our work would be great there! His site handles a lot of biological anomalies. He said his site would be perfect for a new budding department. I read over the site dossier; the equipment and facilities available is just so... tantalizing.

I&#39;ll give him a heads up if you are willing to move. Is your team ready to move to the US? I know Jenny is thrilled, as we&#39;re going to be part of a new department. It&#39;s very funny how it could have been me and her being the founding members for all this groundbreaking work from SCP-6009... her loss, I guess!  

For our name, how about “Nanobiomics”? Short, simple, punchy, and most importantly, never heard of before - just like our work for Project Vigenère.

[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 8, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]

[[&lt;]]
I love it. The site, the name, all of it.

Also, while in China, Dr. Mann asked me if I would be interested in being the department director. Mann warned me about all these responsibilities and hard decisions he had to make... I definitely am not ready yet.

You are far more driven and passionate about the project than I am, and definitely the most brilliant geneticist I&#39;ve ever seen.

So, salutations and congratulations, Director Chiang!
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@



[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
July 29, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Ryo,

You know, I’m not going to lie, I was kind of mad at you for thrusting the director job at me. I&#39;m not ready for that responsibility either! You could have at least asked first!

But now, I forgive all of that. Pope Waley absolves you of all sin. Go, and sin no more.

I forgive you because Site-234 is... well, everything.

The junior researchers here actually know how to properly operate a PET scan without you yelling at them. The sequencers don&#39;t just randomly decide to die without you staring at it every minute. The incubators here don&#39;t even have that growth medium smell! I didn&#39;t think that was even possible. We got our own labs, Ryo. No more sharing refridgerators, no more of my experiment being ruined because some doofus from Xenobiology cross-contaminated my samples.

Also, Sherry and Leep are so nice! They can&#39;t wait to meet you.

This really feels like a reward, man. We&#39;ve made it.

Back to business: this is what Matsunaga and Jenny have been working on for the past few months. I think Matsunaga said that Dr. Mann had a surprising helpful idea with data overload while they chatted in China. I do not understand nanotechnology at all but I think it works well for one very good reason, as you will see.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@

++ **__Second Generation MH Nexus Design: Distributed Nexus Design to Avoid Physical Transduction__**
**Dr. Matsunaga Yuko • Dr. Hsieh Jin-Yan**
[[/=]]

[[&lt;]]
Previous implantation of the Matsunaga-Hsieh Nexus (&#34;MH nexus&#34;) techniques required open-brain surgery, which both has significant risks for patient post-operative recovery and physical damage to 6009-Catena. In addition, the exit port of the first generation MH nexus required physical wiring that both was prone to infection and causes significant discomfort, which typically caused the subject to try and remove the nexus if not properly restrained. The integrity of 6009-Catena is also affected by the physical wiring, though to a previously acceptable degree.

With the invention of a next generation MH nexus coming on the heels of new semiconductor technology, a key component was the ability to allow the elimination of physical wiring. The second generation MH nexus solves the problem by using a distributed Quantum-Frequency Identification network, which spreads the signal out by using 24 nexuses within the brain. Previously unused due to QFID not being able to sustain the 0.4 TB/s transmission speed needed, the distributive load reduces it to only needing 10 GB/s per nexus, thus allowing usage of QFID.

The implantation of the multiple nexuses is subject to current study. 

[[/&lt;]]
[[/div]]

[[/collapsible]]

So, yeah! No more itchy exit port.

You’ll love it too when you get here, Ryo. Or rather, Assistant Director Okami. (Congratulations, salutations, and all that - can’t expect me to handle everything!) I need someone that is reliable by my side. You aren’t escaping the infamous bureaucracy that infects all parts of the SCP Foundation that easily!

It’s so nice. I feel like our hard work has finally paid off.

[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]

[[=]]
[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
August 9, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

I can’t wait to see Site-234! Unfortunately I have to stay behind for another month. Aguinaldo will be there soon though, I’ll get him to bring some of those red bean mochi treats I know you like so much. 

I have to stay behind because I need to finish the observation of some of the D-Class that I was working on; Dr. Mann was in Japan, and came by Site-84 to chat about our new department. The probe that Matsunaga and Jenny were working on? It was actually on his suggestion that he gave them soon after that last symposium. Jenny finally made a breakthrough with from a new paper on  semiconductors that TSMC released, so when they finally came through, Mann wanted to develop a new insertion mechanism with me. We tried some stuff out a week ago, and he finalized his revised procedure. I’m… not especially thrilled about the new procedure, but it’s such a vast improvement over our current one.

As you know, the exit ports for the MH Nexus is the most clunky and error-prone part of an otherwise ingenious product. One, it’s basically an open wound that goes past your meninges. And surgical wounds healing are usually very itchy, so the D-Class would obviously try to scratch it… and well, even though the wire is anchored under the scalp and can’t be removed, pulling on it obviously still affects something.

Mann&#39;s technique is... well, just read for yourself.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@

++ **__The OMNI Procedure: Minimally Invasive Method for MH Nexus Implantation__**
**Dr. Everett Mann • Dr. Okami Ryōsuke**
[[/=]]

[[&lt;]]
The new method, developed in tandem between Dr. Mann and Dr. Okami, utilizes a combination of supraorbital keyhole craniotomy and nasal-entry neuroendoscopy. The Okami-Mann Nexus Insertion Procedure (&#34;OMNI procedure&#34;) is minimally invasive and improves on pre-operative survival. The double-endoscope technique requires a team of two neuroscientifically-trained medical personnel to safely perform.

The subject&#39;s head should be fully restrained to prevent any movement. As this is a form of intraoperative brain mapping, the patient should be sedated and anesthetized but should be awake.

The first endoscope (&#34;detector&#34;) is inserted to the top of the nasal cavity and penetrates past the sphenoid sinus. The endoscope insertion must be angled correctly and reach the hypothalamus, allowing it to monitor levels of adrenocorticotropic hormone (ACTH) and corticotropin-releasing hormone (CRH)[[footnote]] Part of human stress response for cortisol release [[/footnote]] levels. An already-high level of CRH is considered normal during the procedure.

A keyhole is made in the anterior cranial fossa through the eyebrow, allowing a second endoscope (&#34;planter&#34;) to access and perform the insertion. Inserting past the frontal lobe, the keyhole endoscope reaches the postcentral gyrus.

Because of 6009-Catena and its frequent association with outside stimuli, the MH Nexus is best planted within regions of high reactivity. This entails repeated the &#34;planter&#34; releasing weak electrostimulation of the somatosensory and cingulate cortices, both which are involved in nociception[[footnote]] Pain and other negative stimuli [[/footnote]]. Because the patient is sedated, there will be no capacity for physical actions or vocalizations. Any movement may cause misalignment of the endoscope and damage nerves nearby, such as CN I (olfactory) or CN II (optic).

The researcher monitoring the &#34;detector&#34; endoscope for stress levels should inform the researcher operating the &#34;planter&#34; endoscope when to insert one nexus at high levels of CRH and ACTH. The &#34;planter&#34; endoscope currently can only insert up to three at a time; the Nanobiomics Department is currently designing custom types to avoid repeated retraction and insertion from the keyhole.

Post-surgical recovery with an amnestic regimen has proven to be useful in alleviating the memory, but amnestics have also shown to result in unpredictable conformational changes to the neural circuitry. Because of the brief four hour duration of the surgery, the memory of the surgery constitutes a smaller alteration to 6009-Catena than any known amnestic regimen.

Regardless of the surgical safety, all patients report emotional and psychological trauma of the experience. Even though the patients are properly anaesthetized and sedated, subjects uniformly report the experience is deeply unpleasant. A search for non-amnestic alternatives to alleviate the memory is currently under way.
[[/&lt;]]
[[/div]]

[[/collapsible]]


Mann came up with the OMNI acronym, which was very kind of him, but all I really did was ask a lot of stupid questions when we were testing the technique out. Most of my contribution was the note of concern near the end there.

I&#39;ve done plenty of intraoperative brain mappings, but never with two endoscopes at once, and since the D-class is awake you can see them stare at you. Not fun for me, but definitely even less fun for the poor dude on the table.

Amnestics obviously mess with your neural mapping and memory formation, and since that&#39;s so important to 6009-Catena, not being able to use them is really, really unfortunate - maybe that could be another angle of attack in Project Vigenère?

The D-Class obviously hate the surgery, but none of them report the migraines or headaches that were so common from before, and since now there’s no exit port we don’t need to worry about them scratching it. Our psych staff says that there’s no observable behavioral change, which is good news for our study. Allowing them to do other tasks was actually giving us a wider variety of SCP-6009 interactions with the nuclear genome, and also helps since Site-84 has such a limited D-Class quota. They allotted us more people since it&#39;s Dr. Mann requesting it, but Site-84 just don&#39;t have that much housing space.

I’ll be stuck here for the next month or so before I can go to the US because I need to finish the post-operative observation for them and get some pre- and post- operation 6009-Catena mapping in.

I can’t tell you how much I want to leave this dump of a site. I like Yamataka, but I will not miss tiny little 84.

I look forward to seeing you, and our Nanobiomics family soon.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]



[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
August 25, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Good things come to those who wait. You will love it here, Ryo.

Thanks for the treats! One thing I miss in Alabama is that there’s no good restaurants of any Asian variety anywhere… but the steak here is cheap and delicious, so win some lose some I guess.

I&#39;m glad Mann helped you out with the new implantation... woof, that made me uncomfortable to just read, don&#39;t even want to think about it being done.

The probes are important, and the fact that the most irrelevant part is causing the most problems has always annoyed me. I’ve heard it’s itchy as hell around the exit ports, so solving that was nice... this just wasn&#39;t quite the solution I thought of.

Anyways, I’m starting a short paper for VGWAS and ChOMP-Seq with the association of the 6009-Catena map clustering. I was working on a new method of data analysis, and I met with some of the people that are now in our department to refine the model.

A part of the project I’ve been working on was eliminating statistical noise - one big obstacle in Project Vigenère was that after we filter out the regular genes, there’s really not a lot of useful data left - we are dealing with genes and proteins that exist in extremely minuscule amounts but affect a whole lot.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
++ **__Project Vigenère: VGWAS as a Modification for Association Studies__**
Dr. Chiang Wei-Huo • Dr. Hsieh Jin-Yan • Dr. John Batesman • Dr. Louis Graham • Dr. Ali Youngman
[[/=]]

Conventional GWAS studies make use of genetic markers, typically single-nucleotide polymorphisms (SNPs), in association with a disease or other condition. GWAS is useful as a tool due to it being a non-candidate-driven assay that is phenotype-first, as opposed to genotype-first. Project Vigenère data was built on ChOMP-Seq, but conventional GWAS does not take heteroplasmy into account. By including the mitochondrial genome in GWAS and treating certain gene interactions as phenotypes, we utilize a modified version of the Vigenère genome-wide association study (VGWAS).

VGWAS is an extension of conventional GWAS applied to ChOMP-Seq. Additionally, in contrast to the allele frequency given by GWAS, the VGWAS results out interaction frequency. The interaction was obtained from the FMO probe as part of the MH nexus project, which are able to monitor the resonance profile of the interactions; at the same time, the complexity of proteins and the fact that many have similar conformations lead to non-unique resonance profiles.

A preliminary comparison of early results obtained from VGWAS and the standard reference Cambridge genome widely used show slight disrepancies that could be significant. We are currently in development of a standard distribution that could be used to model and discover unusual patterns from Project Vigenère. The construction of a reference mitochondrial distribution, as well as a nuclear genome interaction map, would be the next part of the project. 

[[/div]]

[[/collapsible]]


What do you think? We’ve been finding pretty promising results of certain pathways and genomes - I think the most interesting is that regardless of the D-Class personnel’s race, genetic material, or family history, there always is the high prevalence of mitochondrial DNA that has the MT-A3 gene. Even though most people do have the MT-A3 gene, I only ever see the MT-A3 gene interact with certain chromosomal genes: LRI3, PFO4, CRW2, and a handful of other less common ones.

There’s also certain subclusters that keep popping up - and activate during, say, eating certain foods or doing certain activities. Not sure what&#39;s there. 

And speaking of D-class, have you tried blindfolding them during the surgery? Wouldn&#39;t that also make it less terrifying for them, without seeing two doctors hover over them playing with their brain?
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
September 1, 2012
[[/div]]
[[div]]
[[&gt;]]
Site-84
Fukuoka, Japan
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,

You would think so, but we detect a higher baseline level of stress in all subjects when we tried that. I think it&#39;s because not seeing what&#39;s happening makes it even scarier... I often feel like I&#39;m obligated to keep apologizing during the procedure. Mann is cool as a cucumber, obviously.

All in all, I&#39;m happy to report though: the OMNI procedure allows full recovery, and definitely should be used instead of our original design.

I’m flying in tomorrow. I am so, so happy we can collaborate in person. No more email chains.

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@



[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 8, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Ryo,

I got an unannounced visit from someone representing the Ethics Committee today.

Are you free on the 14th? They want to observe the OMNI Procedure and judge its suitability for use in civilian subjects. Who&#39;s also free that day to partner? I think Micheal is busy that day, so maybe Graham? If you aren’t free, I think DePaul or Raskin can do it.

If they approve, we’ll be able to use the Foundation&#39;s contacts at the University of Alabama Birmingham as a front to collaborate with other universities and research agencies to get a wider pool for testing. We&#39;ve kind of hit a wall with D-Class data, so I&#39;m glad we&#39;re getting some fresh subjects. 

Which, here&#39;s the draft I haven&#39;t finished about D-class... I&#39;d probably just throw it to the new people, honestly, I haven&#39;t been able to look at the data much because my schedule is full of interviews.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
++ **__Project Vigenère: Promising Early Results from VGWAS Clustering__**
Dr. Chiang Wei-Huo • Dr. Okami Ryōsuke • Dr. John Batesman • Dr. David Meng • Dr. Elias Raskin
[[/=]]

Clustering data obtained from VGWAS revealed a preliminary interesting datapoint: a common interaction has been observed common across D-class  personnel. 80% of D-Class personnel exhibit a isomorphic pathway in the hypothalamic attack region, which VGWAS placed the pathway in association with the UP4/MT-F5 interaction. Both the UP4 gene and MT-F5 gene are found commonly in the general public, but MT-F5 expression is inhibited if MT-L2 is expressed. People with the UP4/MT-F5 interaction lack MT-L2 within the mitochondrial genome, and almost uniformly belong to this cluster.

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@


{{tbd: tell john to add the stats stuff}}


@@@@
@@@@
@@@@
@@@@
@@@@


[[/div]]

[[/collapsible]]


It&#39;s not in the paper but we tentatively named it the D cluster because it&#39;s in like, 80% of the D-class data we have, and they have no similar genomic history or background. But the MT-L2 and MT-F5 thing is consistent in almost all of them, which leads to the same UP4 interaction, which results in the same 6009-Catena formation. Very strange if it’s a coincidence, that&#39;s why they&#39;re currently in the D cluster. It&#39;s also usually the more violent ones too, the calmer ones don&#39;t seem to have that genetic interaction. 

I miss doing research, Ryo. Most of my time now is spent dealing with the logistics of collaborations, approving testing, interviews for people that want to transfer into our department, then meet with Leep and Sherry to discuss equipment upgrade... and hey, we&#39;re getting people from other sites interested in being part of Nanobiomics! Ironic, isn&#39;t it, since we only came to Alabama to be one team under the same roof...

I&#39;m glad the field of Nanobiomics is popular, and I&#39;m immensely proud of it, but I&#39;ve been so busy... Project Vigenère is our baby that we don&#39;t control anymore. I&#39;m so proud of every single part of it. 


[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 12, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

No, I didn’t request general public usage, why would I? I do agree that having civilian subjects are going to vastly help expand our scope... at the same time, civilian subjects are usually signing up willingly? So at least it&#39;s less fucked up than what we&#39;ve put these poor D-class through.  

I&#39;m free to showcase OMNI, I&#39;ll take Graham.

I saw they’ve announced the location for the next Biological Sciences Symposium - May 10 to May 17, and it’s in Kyoto this year! Can you believe it was only last year when we had to beg Yamataka for a measly little spot? Now we’ve been invited to a keynote speaker spot. It&#39;s going to be just you and me, representing the new field of Nanobiomics. 

I love Kyoto! I know it very well. People seem to think that it&#39;s just old temples and streets, but it&#39;s more than just that. I usually don&#39;t really like being a tourist in my own country, but just for you, Waley, I&#39;ll show you around. It will be my apology for skipping out in Chengdu.

We&#39;ve been so busy lately! This would offer a nice respite from our work. 
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
April 25, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Ryo,

Sorry I didn&#39;t get back to you sooner, I really need to get some sleep. I forgot about your email, but just now I got a message from EC about the OMNI thing, so, congratulations! I think Leep would be in charge of setting your contracts up.

And yeah! Keynote speakers! I didn&#39;t think it would happen in my wildest dreams.

Here&#39;s something that might shock their pants off. I&#39;ve had this in my back pocket for a while now but this is going to give me the kick to finally finish it.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
++ **__Project Vigenère: Neo-endosymbiotic Theory__**
Dr. Chiang Wei-Huo • Dr. Okami Ryōsuke
[[/=]]

Our hypothesis regarding the origins of SCP-6009: a particularly robust bacterium strain or mitochondrion, which that overtook the original mitochondria in early human evolution. 

If this hypothesis is true, then it is possible that human evolution was driven by SCP-6009. It may be the reason for the speciation and differentiation of early //Homo sapiens// from other hominids. 
@@@@
@@@@
@@@@
@@@@
@@@@


[[/div]]

[[/collapsible]]


This could be a good update to Project Vigenère. I think it&#39;s going to get a lot of people mad because this is a fundamental change of the concept of the &#34;mitochondrial eve&#34;, but I think our theory makes the most sense: a particularly robust bacterium, or mitochondria from some ancient human ancestor, and somehow spread to other humans during our early evolution. 

And seeing how most of the mitochondria&#39;s proteins needed to be coded by nuclear genomes, wouldn&#39;t it make sense for SCP-6009 to maybe have some mechanism to control mutations for other genes in the nucleus, which, essentially, would drive human evolution?

Oh yes, people will get very upset, I can&#39;t wait. Just listen to how grandeur &#34;neo-endosymbiosis&#34; feels. I&#39;m going to look into the molecular clock comparisons that Dr. Kirkoff has been working on, don&#39;t tell him it&#39;s for this. Administrative work is tiring, and I do feel kind of detached from the lab work we used to do. Haven&#39;t written a paper in a while. 

I want you on this one. Just the two of us, a return to basics.
 
@@@@
@@@@

I haven&#39;t taken a vacation in forever. I already know Kyoto is an amazing city, but going there with you will be a trip to remember. 


@@@@
@@@@

[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 1, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]


Ryo,

I still can’t thank you enough for the tour around Kyoto. The symposium was already a smashing success, and that amazing trip was the victory lap that we deserved. You really made me see the world in a new light. We really should do this more often; we&#39;re literally living on the same site, why don&#39;t we get out more? See all what America has to offer! 

Also, on the heels of even more good news: I just got out of a meeting with Leep and Tilda Moose. You’ve heard of her, right? She’s been the long time director of Site-17.

Director Moose has heard our keynote speech about the neo-endosymbiotic theory. She&#39;s very intrigued about and says there&#39;s a highly classified species that she wants Nanobiomics to look into. It&#39;s one of the few extant species that both exhibits near-human intelligence and closely-related evolutionary history. Something Director Moose came across in her early days... she thought that it would be pertinent to the neo-endosymbiotic theory for us to see if a closely related species to us also has SCP-6009 or a similar structure within them. 

But that&#39;s not why she came to visit in person; she came because she said that the Overwatch Council has taken an interest in our department, and wants us to move our base from Site-234 to Site-17. Here at Site-234 we were mostly dealing with microbiology and biochem stuff, stuff that you can safely keep in a petri dish; but Site-17, well, we’ve all heard the legends about what hides in that place. I don’t even know which ones are true. She says we will know once we accept her offer. 

Of course, for our troubles, Director Moose has offered us better compensation, newer machines, higher clearance level, yada yada yada. But most importantly, she says since they’re doing expansions recently, we are getting our own wing in the new building.

I thought Site-234 was swanky, but Site-17… in our own wing… I really don’t know how to respond. I want to get your input as my right hand man, my brother from the start of this wild journey. Leep says he’s really gonna miss us, but, how can I say no? 
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]

[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 3, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]

Waley,

Site-17? Wow, that’s really not what I was expecting.  

I don’t know if we can even afford to say no if the O5 has taken an interest in us.

You’ve always been the one with the big heart. Always being there, not just as one of the most brilliant geneticists I’ve ever seen, but also as a close friend. 

Without you I don’t think our department could grow into the powerhouse that it is today. Can we believe we only first met and started Project Vigenère around two and a half years ago? Feels like it’s been my whole life, this wild ride.

We’ve only been here for… ten months? Wow, it feels so much longer than that. I’m gonna really miss Sherry and Leep. At least our department will still all be together. Our weird little family has grown so big now!

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 8, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Ryo,

Thank you for the kind words. The same goes for you! Without you we would both be still stuck at our home sites doing god-knows-what…

I responded to Director Moose but she said she already approved of the transfer for us, since it&#39;s pretty clear we weren&#39;t going to reject the offer. Honestly, it&#39;s too good to pass on, who wouldn&#39;t want such a promotion? 

They’re still building the new wing, but Director Moose has given me clearance to access the files we will have access to once we get there. You should have clearance too, since you’re Assistant Director. 

There’s some juicy stuff we can do. Just look at what humanoids are there. I feel like a wide-eyed junior researcher, boggled at the most mundane things, and it is refreshingly exciting. 

I&#39;m going to be at Site-17 to iron out some paperwork, and inspect if the new wing fits in with what we need - I&#39;ll be pretty predisposed here since I want to make it a seamless transition, so you might have to take over for a bit.
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]

[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 10, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

We&#39;ve been around for a while, but reading all of this... this is the first time in my entire career, I feel like I&#39;m finally, properly, part of the Foundation. 

I’m reviewing the stuff we have clearance for now. I think [[[SCP-2828]]] would be of interest - a vestigal organ that has antimemetic properties! I have some theories about evolutionary techniques that served to protect us from predators having origins in SCP-6009. Remember the [[[SCP-3966]]] response you worked on that lead to the discovery of SCP-6009? Maybe there&#39;s some genomic basis for protection against those 4-dimensional spiders too! 

After all, we co-evolved, and if the neo-endosymbiotic theory is correct, then this would be one mark for co-evolution. It&#39;s weird how humans are the only species that have mitochondria that acts like SCP-6009... and since Director Moose requested, yeah, now that I can read the file, [[[SCP-1000]]] really would be a great stepping stone. 

Or, now we have a whole new biological paradigm from the view of Nanobiomics, we can look into [[[SCP-1237]]]. It&#39;s genetics and neuroscience, Waley, it&#39;s begging us to study it. And if we can find the genetic and molecular basis for one anomaly...

The Foundation really is a whole treasure trove. 

I&#39;m so happy that we took on this wild journey. 
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 18, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

I have this early unfinished draft that you should review on a hypothesis regarding reality benders - come to think of it, we never tested on humanoid anomalies. And Site-17 is known for having a ton of them! I&#39;ve actually never done a brain scan of humanoid SCPs, it&#39;s going to be fun trying to map the 6009-Catena and see if the clustering matches up with regular humans!

I&#39;ve never met a reality bender before, apparently there&#39;s [http://www.scpwiki.com/clef101 some training] that we need to do before we are allowed to interact with them. I&#39;ve already signed up to start by the time we get to Site-17, do you want me to sign you up too?

Sorry, I&#39;m getting ahead of myself. 
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

[[div class=&#34;emailchiang&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
June 29, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-17
Nevada
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Ryo,

I&#39;m so happy to see you this excited! The Foundation really is not sparing any expense for our pending arrival. I&#39;ve been meeting with all these department heads that want to get on our good graces! All these fields and departments that I didn&#39;t dream to ever interact with. I met more people here this week than my entire four years in Site-168, truth be told. 

And yes, sign me up for the reality bender seminar. There&#39;s real power and danger in these sites. The other day I had to partake in a breach drill, and they take it very seriously here. No more sleeping through drills, Ryo! 

At any rate, one of the new things Director Moose showed me was this system that Michael and Matsunaga have requested. Apparently I approved this? I don&#39;t really remember this, but the design is pretty ingenious. Basically, it takes in ChOMP-Seq data and tries to find the best fit in the clustering data we got from VGWAS, so we can have a pre-projection of how the 6009-Catena region would look like before hodological mapping.

I thought it would be fun to see my own profile. Remember when we were testing early beta versions of ChOMP-Seq, and I had to use my own DNA because we ran out of test subjects? That felt like an eternity ago. My genome is still in the Project Vigenère folders so I threw it in just for the sake of it. Obviously we still have a lot of work to do, but my profile so far has GPK2/MT-UE4, ABCC11/MT-FR2, O-M122 haplogroup, which puts me in the civillian E-38 &#34;passionate&#34; and E-20 &#34;curious&#34; clusters. I could tell Director Moose was a bit skeptical, so I assured her that it&#39;s only inaccurate because we haven&#39;t properly attuned it yet. I told her, hopefully in the future, we&#39;ll be able to associate the genomic profiles with psychological profiles better. 

This would be pretty useful in predicting which psychological cluster people would end up in, without needing the OMNI procedure! Obviously there&#39;s still a lot of honing to do, but we might be able to retire that awful technique soon. 

I&#39;m sorry I&#39;m not back yet - I miss you! Please hold down the fort for me. I trust your judgement to do the best for the Nanobiomics Department! 
[[/&lt;]]
[[&gt;]]
Waley
[[/&gt;]]
[[/div]]




[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
July 7, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

Aww, I miss you too! 

I really would like to come by and see for myself, but I have to sort through a lot of paperwork for personnel transfer. When did we get more than a hundred people under our department? 

Sometimes I feel so lucky that I was assigned to work on Project Vigenère. 

You and I, we come from very mundane scientific backgrounds. Nobody balks at neuroscience or genetics. What we&#39;ve been doing have been circling normal sciences. No paratech, no elder gods, no secret society. I&#39;ve heard stories about the magical going-ons in other parts of the Foundation, and I envied them. 

These tools we created is our own brand of magic. I can&#39;t go back to the old days, now I know what lies within our own brains, just wriggling away. What are they trying to do? How are they doing it? SCP-6009&#39;s base anomaly is plain, but we dug deep trying to find out what gave it that strange edge, and we emerged with a whole new paradigm. 

We are going to change the world with our project, I can feel it. 

Still, we all miss you here at Site-234! I need you here with me. Sherry has been asking about you. Where are you?

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
July 13, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

Someone from the Ethics Committee came by to inform us of guidelines dealing with fetal subjects, because it seems like we will be getting clearance for that now? 

This is going to be interesting... I don&#39;t imagine the OMNI procedure can be done on fetuses or infants, and I would not subject a baby to that. If we can find a way to make it less traumatizing it could be a great way to finally get confirmation on how 6009-Catena is formed in the first place, before the brain is fully developed!

I really need you back here, collaboration and transfer requests are skyrocketing because the word has spread about us moving to Site-17. 

We miss you here! 
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]

[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
July 24, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
@@ @@
[[&lt;]]
Waley,

Director Glass from the Psychology Department came by yesterday looking for you, he says he would really like to initiate a large scale collaboration request. He&#39;s been told about our labeling inaccuracies and has had people looking into the way we are clustering - he says it&#39;s painfully obvious none of us study psychology, and that our current cluster labels are about as accurate as a horoscope.

I didn&#39;t want to go over your head on this one, since Director Glass is kind of a big deal, and he specifically requested to speak to you. I told him that you&#39;re currently predisposed, but when they ask me when you&#39;ll return, I don&#39;t have a good answer.

Did something happen in Site-17? It&#39;s been like, two months now, how much can there be to sort out? I really hope you&#39;re safe, I&#39;m getting worried. 

Please respond at your earliest convenience.
[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]


[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
August 10, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Waley,


@@@@
@@@@

Did you get my last email?

@@@@
@@@@

Please respond at your earliest convenience.

[[/&lt;]]
[[&gt;]]
Okami
[[/&gt;]]
[[/div]]
[[/=]]

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@

[[div class=&#34;emailokami&#34;]]
[[=]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
August 17, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-234
Alabama
[[/&gt;]]
[[/div]]
[[/div]]
[[/=]]
[[&lt;]]
Dear Waley,

If I did anything to upset you, please forgive me. I&#39;d like to think I&#39;ve been nothing but cordial to you, but I don&#39;t know why you have been ignoring me. 

I know you&#39;re still alive, because the Foundation generally don&#39;t keep that a secret. A dead director has got to be replaced as soon as possible to keep things running. I know you&#39;re not transferred out, or got promoted, or got demoted, or any of that. So the only reason for radio silence is either Site-17 got wiped off the earth and no one found out, or you are ignoring me deliberately. 

We need you here, Waley. I wouldn&#39;t say the Nanobiomics Department is a mess, but it certainly isn&#39;t the same without you. You complain that you haven&#39;t been in a lab in a while, but you are still our project leader, and you have big picture insight into how to proceed with Project Vigenère that no one else has. Just because you don&#39;t need to be tinkling away with equipment and charts doesn&#39;t mean your contributions are unimportant. You&#39;re still a brilliant geneticist, and no amount of Foundation bureaucracy will take that away. 

It&#39;s just not the department that needs you, but I need you too. When I took on this project, I was skeptical and shy. I was borderline emotionless. I was not adventurous, and I rarely got excited for things. I know I am kind and caring, but I don&#39;t really like to show it. You opened my mind, not just in scientific inquiry, but awoken aspects of myself I didn&#39;t even know was there. 

I don&#39;t think I had a friend as close as you, Waley. And I&#39;d hate to lose you. Or at least, have the decency to tell me why you&#39;ve decided to leave. 

You&#39;re not just our director, Waley, you are probably the only person that understands SCP-6009 better than me at this department. We need you as our captain. 

Please come back. 

The Nanobiomics Department needs you. 

I need you. 
[[/&lt;]]
[[&gt;]]
Ryo
[[/&gt;]]
[[/div]]

@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@




[[div style=&#34;border:solid 1px #999999; background:#f2f2c2; padding:5px; margin-bottom: 10px;&#34;]]
[[=]]
@@@@
++++ NOTICE FROM THE FOUNDATION RECORDS AND INFORMATION SECURITY ADMINISTRATION
@@@@
The following files have been flagged by the RAISA database for needing review for the reason ##orangered|**{{Inaccurate Personnel Status}}**##

For the reason of ##orangered|**{{Department Director Request}}**##, the file(s) selected should be {{##orangered|__**EXPUNGED**__##}}
@@@@
@@@@
@@@@
[[/=]]

[[div class=&#34;emailchiang&#34;]]
[[div style=&#34;display: flex; justify-content: space-between;&#34;]]
[[div]]
August 20, 2013
[[/div]]
[[div]]
[[&gt;]]
Site-17
Nevada
[[/&gt;]]
[[/div]]
[[/div]]

[[&lt;]]



Ryo,

I’m sorry I’ve been gone for so long, I&#39;m safe, don&#39;t worry. 

You did nothing wrong, and the apology should be mine. I was at Site-17 for quite some time, yes, and I kept neglecting your emails. I&#39;ve read all the them, and I&#39;m glad you&#39;re excited! I just couldn&#39;t bring myself to tell you what&#39;s on my mind. Every day, I would watch as the construction teams build up something new, read over the new quotas we now have, think about how to allocate funding... 

As for Site-17 itself... walking in the halls, you can sense the secrets hiding behind the walls. It gave me a chilling sort of pride, like you&#39;re walking inside history itself.

I... didn&#39;t know how to word my thoughts, and I really didn&#39;t want to hurt your feelings. The more excited you were the more it hurts for me to think about what I&#39;m about to tell you. You&#39;re my closest friend and confidant, so I want you to be the first to know:

I will be resigning as director, and transferring out of our department.

It seems natural since you threw this hot potato of a title at me, I’m throwing it back to you now. Following my transfer, you, as the Assistant Director, would naturally take over as Director of Nanobiomics. I don’t think anyone would complain, everyone loves you at Nanobiomics. Salutations and congratulations, Director Okami!

We really built up this whole department from nothing. You and I, from two underfunded sites, are now growing this project to an unimaginable scale. Even the name &#34;Nanobiomics&#34; commands respect.

I&#39;m really proud of everyone in our department - Jenny, Matsunaga, Michael, even the new people - but I want to thank you specifically, Ryo. And I remember your old self, yes. I remember like it was yesterday: awkward, overtly formal, only here because you were assigned, and honestly kind of skeptical of our project. The Dr. Okami from three years ago and the Ryo I know now could not be more different. 

Despite the grandeur of what we built up for Nanobiomics, SCP-6009 itself is mundane; all the intrigue that we found in our department was the work of our own two hands. Project Vigenère already surpassed its original goals a long time ago. The core anomaly is well understood. As the project grew, the creative sparks flew and from the tiny seed grew this behemoth that we stand on. ChOMP-Seq, OMNI, the MH nexus… wow, Project Vigenère really gave us a lot, didn&#39;t it? There&#39;s people in Nanobiomics working on things I don&#39;t even understand anymore. We&#39;ve unknowingly crossed a threshold somewhere.

The last email I sent you, I mentioned something that Director Moose showed me, the terminal that they&#39;re setting up. They didn&#39;t even name it yet, but I told you the gist of what it does: We can get how the neural connections worked, and categorize the psychological profiles according to the mitochondrial genetic data. 

Did anything strike you as disturbing in that sentence? 

I imagine you didn&#39;t really think much when you read it. I didn&#39;t think much of what I told Director Moose either. Why, yes, we associate genomic profile with psychological profiles! What&#39;s so weird about that? It&#39;s the natural next step of what we&#39;re doing. I was really impressed of the design! I was thinking what we could even do further... then it hit me. 

I can already envision it: just take a sample, run it through the model, and in a day or two the sleek terminal will pop out your subject&#39;s psychological profile and likely decision making. No need to get permission to rack the brains of every poor sap that the Foundation decides to snatch up. And it works best for non-anomalous humans, just everyday folks like us! Well, now we’re going to be at Site-17, we might find out if it actually works on humanoid anomalies. That&#39;s definitely going to be fun for the whole family!  

And since we have been doing stimulus testing on how 6009-Catena changes neural wiring, Jenny actually sent me plans for the third generation prototype of the MH nexus: if her schema works, it would be able to physically change 6009-Catena connections, directly through a new version of the inserted CMH probe. I don’t think she even realized the implications of what she is creating. 

I haven’t been able to sleep since I realized what we&#39;ve discovered in Project Vigenère. I have so many conflicted ideas what might be the endpoint. We even dangled the prospect of understanding the evolutionary tactics used by SCP-6009 in front of them in the most recent symposium...

It&#39;s not perfect yet, but it&#39;s serviceable enough for me to be worried. Furthermore, what does this mean for how personalities are shaped, if it was always just from a genetic basis? 

I&#39;m going to tentatively call my position &#34;genetic determinism&#34; - a person having certain types of genomic interaction can be categorized into certain cluster of behaviors. I know this is uncharacterisitcally nihilistic, Ryo, and I would understand if this makes you worried for my sanity, but I&#39;ve had a lot of time to think, and I just can&#39;t reason my way out of this. 

During these sleepless nights, I mostly just drive out into the Nevada desert, and look into the skies out here. We&#39;re both city people, Ryo, and we really are missing out. Stars are great philosophical companions. 

I tried to convince myself that&#39;s not true. C&#39;mon, our brains don&#39;t stay the same our whole lives, that&#39;s Neuroscience 101. You&#39;re not just... born with what you&#39;re going to be like. Your personality is shaped from your joys, memories, traumas, all that! The outside world changes you! Nature versus nurture! 

Is personal growth just a joke then? Are people just destined to be a certain way, hard-coded in their genes? 

I keep telling myself, there&#39;s no possible way that the intricacies of the human mind could be broken down into chemicals and numbers in a machine like that. But I know what we&#39;re studying, Ryo, and so do you. We found this hidden library underneath the dense genetics code, and it&#39;s only a matter of time and effort to see how long we can decipher what&#39;s in this library. 

Every day, I feel watched. I’m suddenly acutely aware of anything I do. It’s like my own mitochondria have betrayed me, which I am aware is an utterly ridiculous idea.  

Perhaps I&#39;m overthinking it. There are so, so many hurdles to overcome. Maybe we already hit the wall of what we can decipher, and the broadest strokes are the extent. Or maybe I’m just paranoid about nothing, inflating the importance of our department, and ultimately nothing important will come of Project Vigenère.

As much as it hurts to think about, I thought about shuttering the department. I have (well, had) the power to make everyone just stop. Ban all research, bury our data, dismantle the department, and we all go back to our old lives. Forget this all happened, another project lost to the Foundation’s annals of time. Maybe even throw this to some MTF with a webcrawler to ensure no civilian ever gets close to the truth either. Tell Director Moose that we&#39;re sorry the Foundation spent so much money on their new wing, pack our bags, and just leave America forever. I actually planned on sending a letter to the Overwatch Council recommending this line of action, but then realized they likely would never respond in the way I want. The Foundation has invested so much in our department at this point, and now I know why. 

I am foolish, Ryo. I really should have seen this sooner. Moose, Mann, Andrews, Yamataka... Why have all these higher-ups taken such a keen interest in our project?  It’s not for the betterment of science, out of the pure kindness of their hearts, that I can tell you. 

I don&#39;t think they have malicious intent as individuals... but I can&#39;t say the same about them acting collectively as the Foundation. They might not know the science behind everything, but they know a good opportunity when they smell one. 

We think we are rewarded for our hard work, but really, it&#39;s only because the Foundation finds us useful. If our findings gave the Foundation no benefit, would they be investing so much into our project? 

It’s really kind of funny - do you remember my original proposal before it&#39;s even Project Vigenère? I listed two goals: investigate how these pesky mitochondria keep moving out and about, and find out how it’s making this weird web in our brain. Well, we certainly answered that second part, but to this day no one really has a clue about how it moves around on its own, and it really is too unimportant for anyone to care. The Overseers are not going to be wasting money for us to find out how why they wriggle around. 

We&#39;ve not only discovered something that shakes the very basis of human understanding, but we also developed a tool that can exploit this knowledge. 

You&#39;re a stronger man than I am. You have the courage to see Project Vigenère to its end. I have conflicted feelings about this incredibly valuable tool. I&#39;m sure the Foundation will find its use for the betterment of humanity. 

I don&#39;t think anyone involved in our quick ascent ever had malicious intent. But I also know we, as an organization, have had a long litany of sins.

I&#39;m sure a few years down the line, even my current moral quandary could have been predicted.

You and I had a beautiful time building up this empire. I deeply, deeply appreciate our friendship. The Kyoto trip will forever be one of my most cherished memories. We really should have taken more breaks from work and explore America together, but just being with you makes work feel less like a burden. 

I really hope you will forgive me one day. 

I do care about you, Ryo, not just as a colleague, but as a friend. 

@@@@
@@@@
@@@@

Thank you for making boring my life a little brighter.

[[collapsible show=&#34;Attached File&#34; hide=&#34;Opening...&#34;]]

[[div class=&#34;blockquote&#34;]]
[[=]]
@@@@
@@@@
[[image http://scp-wiki.wdfiles.com/local--files/scp-6009/kamoriver.jpg]]

{{Kamo River, Kyoto, 2012}}

[[/=]]



[[/div]]

[[/collapsible]]


@@@@
@@@@
@@@@

I&#39;ve led the way to the Promised Land, and I am fine watching from the mountaintops. 

I don&#39;t regret my time at Nanobiomics. But I can&#39;t continue down this path. 


@@@@
@@@@
@@@@


I hope you will maintain contact, but I would understand if you don’t. Or can’t. 

Please stay safe in Site-17, for my sake. Once again, congratulations. And good luck.
@@@@
@@@@
@@@@
@@@@
@@@@
@@@@
Some boxes are better left unopened.
@@@@
@@@@
[[/&lt;]]
[[&gt;]]

Forever your friend,


Waley Chiang
@@@@
@@@@
[[/&gt;]]
[[/div]]
@@@@
@@@@
&gt; **Correction**: Dr. Chiang Wei-Huo has been transferred back to the Nanobiomics Department. 
&gt; 
&gt; The Ethics Committee has requested Director Okami Ryōsuke to enroll Dr. Chiang in a mandated amnestic corrective regimen in order for him to continue his work. 
&gt; 
&gt; Director Okami has been requested to respond at his earliest convenience. 

[[/div]]
@@@@
@@@@
@@@@

The current state of Project Vigenère is not yet sufficient for SCP-6009 to be reclassified as Thaumiel.


@@@@
@@@@
@@@@
@@@@

[[div class=&#34;footer-wikiwalk-nav&#34;]]
[[=]]
&lt;&lt; [[[SCP-6008]]] | SCP-6009 | [[[SCP-6010]]] &gt;&gt;
[[/=]]
[[/div]]

[[include :scp-wiki:component:license-box]]
=====
&gt; **Filename:** main.jpg
&gt; **Author:** 乌拉跨氪
&gt; **License:** CC-BY-SA-4.0
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:Motor_neurons_%E8%BF%90%E5%8A%A8%E7%A5%9E%E7%BB%8F%E5%85%834.JPG


&gt; **Filename:** ladder
&gt; **Author:** Louisa Howard
&gt; **License:** Public Domain
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:Mitochondria_-_TEM.jpg


&gt; **Filename:** mito1
&gt; **Author:** Ivo Kruusamägi
&gt; **License:** CC-BY-SA-4.0
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:PDNA_transfer_via_PepFect14.jpg


&gt; **Filename:** mito2
&gt; **Author:** Louisa Howard
&gt; **License:** Public Domain
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:Mitochondria,_mammalian_lung_-_TEM.jpg

&gt; **Filename:** nexus
&gt; **Author:** Hellerhoff
&gt; **License:** CC-BY-SA-3.0
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:Tiefe_Hirnstimulation_-_Sonden_RoeSchaedel_ap.jpg

&gt; **Filename:** mapping
&gt; **Author:** Joreth
&gt; **License:** CC-BY-SA-3.0
&gt; **Source Link:** http://scp-wiki.wdfiles.com/local--files/scp-6009/mapping.jpg
&gt; **Derivative of:** Connectome_extraction_procedure.jpg, CT_of_a_normal_brain,_sagittal_20.png

&gt; **Filename:** Connectome_extraction_procedure.jpg
&gt; **Author:** Hagmann P, Cammoun L, Gigandet X, Meuli R, Honey CJ, et al
&gt; **License:** CC-BY-SA-3.0
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:Connectome_extraction_procedure.jpg

&gt; **Filename:** CT_of_a_normal_brain,_sagittal_20.png
&gt; **Author:** Mikael Häggström
&gt; **License:** CC0
&gt; **Source Link:** https://commons.wikimedia.org/wiki/File:CT_of_a_normal_brain,_sagittal_20.png

&gt; **Filename:** kamoriver.jpg
&gt; **Name:** Kamo River taken from Shijo Ohashi Bridge
&gt; **Author:** Izu navi
&gt; **License:** CC-BY 2.0
&gt; **Source Link:** https://www.flickr.com/photos/126057645@N04/14727063733

=====
[[include :scp-wiki:component:license-box-end]]</textarea></pre>
  </div>

</div>

<div id="footer">
  
    <em>Last updated 2022-05-18 15:46:27.296194.</em>
  
</div>
</body>